addons/web/static/src/webclient/user_menu/user_menu.js

```diff
@@ -1,28 +1,14 @@
 /** @odoo-module **/
 
 import { browser } from "@web/core/browser/browser";
-import { DropdownItem } from "@web/core/dropdown/dropdown_item";
+import { MenuItem } from "@web/webclient/navbar/navbar";
```

Comment:
It feels weird having to import a MenuItem from navbar/navbar for a UserMenu that does not look - at first glance - directly related to the Navbar (from a file structure perspective).

Reply:
(just wondering if we should rethink a bit the way files are structured under src/webclient)